pkgs/development/libraries/spglib/default.nix

```diff
@@ -8,7 +8,7 @@ stdenv.mkDerivation rec {
     owner = "spglib";
     repo = "spglib";
     rev = "v${version}";
```

Comment:
```suggestion
    rev = "refs/tags/v${version}";
```

Reply:
Why?

Reply:
It's generally better to explicitly mention that it is a tag, else  if there is a branch with the same name in upstream, it would fetch from that instead, creating unstable hashes and unreproducibility.

For this problem particularly though, I am not sure how the hash have changed in a tag, perhaps it was force pushed and recreated?

Reply:
Thanks! That makes sense. However, that was not the problem here. I think they simply moved the tag.